models/example/stg_order_items.sql

```diff
@@ -0,0 +1 @@
+select id as order_items_id from {{ ref('order_items_upload') }}
```

Comment:
so when we built out the `stg_customers` table, we only added the single field for `id as customer_id` to show how you can test that single column. most models, in order to be more useful, include most if not all of the columns in the table.

like you can see in [this model](https://github.com/fishtown-analytics/internal-analytics/blob/master/models/staging/trello/stg_trello_boards.sql) from our internal analytics, we normally format like this and list the columns from the table. we do any renaming, casting, etc in this base model.

can you add the additional fields from the uploads for items, orders and customers for this PR? you do not need to use the `source` way listed in the model linked above - feel free to just use your `ref`